src/backend_model_instance.h

```diff
@@ -84,11 +84,14 @@ class TritonModelInstance {
     const std::size_t hash_;
   };
 
-  static Status SetInstances(
-      TritonModel* model,
-      const triton::common::BackendCmdlineConfigMap& backend_cmdline_config_map,
-      const triton::common::HostPolicyCmdlineConfigMap& host_policy_map,
-      const inference::ModelConfig& model_config);
+  static Status CreateInstance(
+      TritonModel* model, const std::string& name, const Signature& signature,
+      TRITONSERVER_InstanceGroupKind kind, int32_t device_id,
+      const std::vector<std::string>& profile_names, const bool passive,
+      const std::string& host_policy_name,
+      const inference::ModelRateLimiter& rate_limiter_config,
+      const std::vector<SecondaryDevice>& secondary_devices,
+      std::shared_ptr<TritonModelInstance>* triton_model_instance);
```

Comment:
There is already a `CreateInstance()` static function. Is this a duplication?

Reply:
If not, please avoid confusing naming.

Reply:
Yes. Updated: [Add more descriptions and improve function naming](https://github.com/triton-inference-server/core/pull/207/commits/686ebc95701d383515300cbf784fbba546cb9aef)